src/webnn_native/openvino/ienn/src/transpose_sinking.cpp

```diff
@@ -0,0 +1,447 @@
+/*****************************************************************************
+ * Copyright (C) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
As we add this file, please add OpenVINO license into https://github.com/webmachinelearning/webnn-native/blob/main/NOTICE